lib/rules/camelcase.js

```diff
@@ -229,6 +234,18 @@ module.exports = {
                         report(node);
                     }
 
+                // Check if it's a global variable
+                } else if (Boolean(node.scope) && node.scope.type === "global") {
```

Comment:
I copied this from `no-var.js`, I have no idea if it works and how to test locally. Please help me with this.